tagesPlan27_09_23.json

```
@@ -1,12 +1,23 @@
 [
-    {
-    "Kapitän": " ",
-    "Vorname Nachname": "Jules",
-    "Stand Up Datum": "27.09.2023",
-    "Intakevorbereitungen": "testCases Automation mit C# und C++ ",
-    "Intake Übungslaufe mit": "N/A",
-    "Zerfizierung und Deadline": "Automotive Softwaretest Prüfung vorausschtlich am 31.12.23",
-    "Sonnstiges": "CI/CD und Jenkins"
-    }
+{
+"Kapitän": " ",
+"Vorname Nachname": "Petru Walter",
+"Stand Up Datum": "27.09.2023",
+"Intakevorbereitungen": "JSON, Jenkins, Git, GitHub",
+"Intake Übungslaufe mit": "N/A",
+"Zerfizierung und Deadline": "N/A",
```

test11
```suggestion
"Zerfizierung und Deadline": "Nfgf
```suggestion
"Zerfizierung und Deadline": "N/A",
```/A",
```